### ISSUE TITLE
`PhTracedLocator` is confusing name

### ISSUE THREAD

**Issue by levBagryansky** [objectionary/eo#3047]
As for me `PhTracedLocator` is an inappropriate because there are such concepts like `PhLocated` and `locator` so `PhTracedLocator` is confusing. 
Let's rename it somehow. for example `PhTraced`.
@maxonfjvipon WDYT

**Comment by maxonfjvipon** [comment]
@levBagryansky I agree, let's rename it. Offer your version in corresponding PR and we'll see
